### PR TITLE
B 18913 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Given I am a Service Counselor
When i conduct a search for a move or customer
and i dont see the member i am looking for
then i willl see a button to create a new member in milmove

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office app
2. Login as a Counselor
3. Click Search tab
4. Complete a search
5. Observer add Customer Button at bottom of page

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

## Screenshots

image

### CODE REVIEW

**Review by traskowskycaci — APPROVED** [review]
Looks good!

File: src/pages/Office/CustomerOnboarding/CustomerName.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    </GridContainer>
+  );
+};
+
+CustomerName.propTypes = {
+  updateServiceMember: PropTypes.func.isRequired,
+  serviceMember: ServiceMemberShape.isRequired,
+};
+
+export default CustomerName;
```
Comment (danieljordan-caci) [inline src/pages/Office/CustomerOnboarding/CustomerName.jsx:79]: Tests plz

File: src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx [hunk 2]
```diff
@@ -301,6 +308,16 @@ const ServicesCounselingQueue = () => {
...
+        {searchHappened && counselorMoveCreateFeatureFlag && (
+          <Button
+            secondary={false}
+            type="Add Customer"
+            style={{ maxWidth: '225px', alignSelf: 'flex-start', position: 'absolute', bottom: 35 }}
+            onClick={handleAddCustomerClick}
+          >
+            Add Customer
+          </Button>
+        )}
```
Comment (danieljordan-caci) [inline src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:320]: Can you add a unit test for this lil' guy?

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

File: src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx (hunk 2 above)
Comment (deandreJones) [reply src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:320]: added

File: src/pages/Office/CustomerOnboarding/CustomerName.jsx (hunk 1 above)
Comment (deandreJones) [reply src/pages/Office/CustomerOnboarding/CustomerName.jsx:79]: added

File: src/constants/routes.js [hunk 3]
```diff
@@ -80,6 +80,7 @@ export const servicesCounselingRoutes = {
   SHIPMENT_REVIEW_PATH: 'shipments/:shipmentId/document-review',
   BASE_REVIEW_SHIPMENT_WEIGHTS_PATH: `${BASE_COUNSELING_MOVE_PATH}/review-shipment-weights`,
   REVIEW_SHIPMENT_WEIGHTS_PATH: 'review-shipment-weights',
+  CUSTOMER_NAME_PATH: '/Onboarding/customerName',
```
Comment (danieljordan-caci) [inline src/constants/routes.js:83]: Can you change this to `onboarding` instead of `Onboarding`?

Comment (deandreJones) [reply src/constants/routes.js]: just for you

**Review by danieljordan-caci — APPROVED** [review]
LGTM! Thanks for the tests.

**Pull request merged**